lib/howl/commands/app_commands.moon

```diff
@@ -302,36 +313,54 @@ command.register
 -----------------------------------------------------------------------
 
 launch_cmd = (working_directory, cmd) ->
-  shell = howl.sys.env.SHELL or '/bin/sh'
-  p = Process {
-    :cmd,
-    :shell,
-    read_stdout: true,
-    read_stderr: true,
-    working_directory: working_directory,
-  }
-
-  buffer = ProcessBuffer p
-  editor = app\add_buffer buffer
-  editor.cursor\eof!
-  buffer\pump!
+  dispatch.launch ->
```

Comment:
Perhaps all handlers should be invoked using dispatch instead? It's slightly more inefficient to use coroutines for all handlers, on the other hand it means that handlers needn't worry about it. So there's long running commands now, but in the future it will also include any handler that fetches something from the network, etc. Or would that cause other problems?